zabbix/info.json

```diff
@@ -631,6 +703,16 @@
           "visible": true,
           "editable": true
         },
+        {
+          "title": "Fetch Related Alerts",
```

Comment:
Suggestion for Title: Fetch Associated Alerts. You can take help from Infodev team if required.